package.json

```diff
@@ -46,6 +46,7 @@
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
     "ember-fastboot-addon-tests": "^0.4.0",
+    "ember-in-element-polyfill": "^1.0.0",
```

Comment:
I think this needs to be a dependency not a dev dependency (since it is needed for our own `addon/**/*.hbs` files)

Reply:
Cool. I'm try that. In CI, I'm also seeing:
```
error ember-in-element-polyfill@1.0.0: The engine "node" is incompatible with this module. Expected version "10.* || >= 12". Got "6.17.1"
```
https://travis-ci.org/github/ronco/ember-cli-head/jobs/680782963